api/server.py

```diff
@@ -24,10 +24,12 @@
 app.config["SECRET_KEY"] = "37Y,=i9.,U3RxTx92@9j9Z[}"
 Session(app)
 
-if REACT_PORT is not 80:
-    CORS(app, supports_credentials=True, resources={r"/*": {"origins": "http://localhost:{}".format(REACT_PORT)}})
-else:
-    CORS(app, supports_credentials=True, resources={r"/*": {"origins": "http://localhost"}})
+#TODO: Fix CORS origins specification
```

Comment:
Could you also add an issue for this in this repo?

Reply:
I think that's something where the non-existence of the coder ownership comes in :smile: